### DISCUSSION TITLE
Losing Meetings Page Link Each Night

### DISCUSSION THREAD

**Discussion by AkronAAIT** [code4recovery/12-step-meeting-list#1130] [Q&A] [answered]
Each night the link to our Meetings page is lost. This started with the latest update to the 12-step-meeting-list. Deactivating and reactivating the plugin seems to work, Resaving our permalinks also works. However this has had to be done each day since the latest update. I have attempted to deactivate other plugins I thought might be conflicting to no avail. We are on the latest version of WordPress and have all plugins up-to-date.

**Comment by joshreisner** [comment]
hm, sorry about that! we're investigating the possibility that there's an incompatibility with certain plugins. is the page in question https://akronaa.org/meetings/?

**Comment by AkronAAIT** [comment]
Yes. I happened to be working late tonight and had the page up about 10 pm.
I noticed it lost the link around 10:20. I had wondered when it was
happening so I will take a look tomorrow to see if one of our backups is
running around that time. I tried turning off YostSEO yesterday but that
didn't work.

**Reply by joshreisner** [reply]
ok that's good intel, thanks! we are thinking it is a cron, and were thinking it might be either yoast or woocommerce. the other site it was happening to we rolled back to the previous version of the plugin, and should know within about an hour whether that worked or not

**Comment by AkronAAIT** [comment]
I also tried switching between the Legacy and TSML one of the days.

**Comment by AkronAAIT** [comment]
We use both yoast and woocommerce on the site, both are up-to-date and we
are on WordPress 6.2 right now.

**Comment by joshreisner** [comment]
so our best thinking is that a change in `3.14.14` is not compatible with one of those plugins, but we're having a hard time figuring out which. we've tried running the crons that these plugins run and can't trigger the issue manually 

interestingly the other site experiences the issue around the time yours does, but slightly later. ~10:40 CT

we are going to release a new plugin version soon but we need to test some things. the code we're going to release is this: https://github.com/code4recovery/12-step-meeting-list/pull/1131

so to fix the issue locally one option is you could install the wp-rollback plugin, and roll back to `3.14.13` (should not cause any problems) and turn off automatic plugin updates (if they are on) and wait for `3.14.15`, should be out in a few days.

or if you want to be adventurous you could replace the contents of `wp-plugins/12-step-meeting-list/includes/functions.php` with [this file](https://raw.githubusercontent.com/code4recovery/12-step-meeting-list/29ab40c7506a3924d09bec0631c0cecc2fc9f5e6/includes/variables.php)

**Reply by joshreisner** [reply, marked as answer]
@AkronAAIT fyi `3.14.15` is out now and should hopefully resolve the issue for you. while we never determined the cause, deploying it to the other affected site appears to have resolved it for them

**Comment by AkronAAIT** [comment]
Thank you. I installed wp-rollback and went to 3.14.13. I will see if that
helps tonight. FYI, I first tried replacing the functions file with the one
you provided and it was throwing an error in init where the custom post
types are registered.

**Reply by joshreisner** [reply]
did you catch what the error was? i don't see one locally

**Comment by AkronAAIT** [comment]
I had pasted the code in the plug-in editor. When I hit save it said there
was a fatal error on line 7 of the includes/init file so it wouldn't let me
save. There was a traceback but I didn't pay attention since I just went on
to use the rollback plugin. I suppose I could do it again if you think that
would help trace the initial problem.

**Reply by joshreisner** [reply]
oops, i see the problem. above i told you `functions.php` when i had meant `variables.php`

no need to retry, thanks. let's see if rolling back the plugin prevents the reset
